utils/config-utils.js

```diff
@@ -0,0 +1,10 @@
+// `CUTTLE_ENV` is needed to test prod specific files without actually putting the app in to
+// a production state, which requires an actual database connection
+const isProd =
+  process.env.CUTTLE_ENV !== undefined
+    ? process.env.CUTTLE_ENV === 'production'
+    : process.env.NODE_ENV === 'production';
+
+module.exports = {
+  isProd,
+};
```

Comment:
@itsalaidbacklife this won't work with vite, but may work with cypress in which case we should move this check over there instead of in a global config util.